src/fleet/azext_fleet/custom.py

```diff
@@ -198,14 +212,15 @@ def update_fleet_member(cmd,
                         resource_group_name,
                         name,
                         fleet_name,
-                        update_group=None):
+                        update_group=None,
+                        no_wait=False):
```

Comment:
Is this correct?  I think patch update fleet member is always sync, so no-wait is not applicable here?

Reply:
0815 it became async. The SDK member update operations return a poller.

Reply:
https://github.com/Azure/azure-cli-extensions/blob/99f22f3ea687b4afaa97e2da9268dc731cf03b19/src/fleet/azext_fleet/vendored_sdks/v2023_08_15_preview/aio/operations/_fleet_members_operations.py#L594